processor/README.md

```diff
@@ -194,7 +194,28 @@ examples on using the processor.
 <FILL ME IN - I'M LONELY!>
 
 ## <a name="queued"></a>Queued Processor
-<FILL ME IN - I'M LONELY!>
+The queued processor uses a ring buffer queue to relay trace data from the
+receiver or previous processor to the next span processor. On
+receiving trace data, the queued processor produces items to the queue. At the
+same time, the queued processor has one or more workers which consume the trace
+data in the queue by sending them to the next span processor.
```

Comment:
maybe add some description about retry logic?

Reply:
Updated. Thanks for your feedback.